src/accommodation/accommodation.service.ts

```diff
@@ -0,0 +1,117 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { PrismaService } from 'src/prisma/prisma.service';
+
+@Injectable()
+export class AccommodationService {
+  constructor(private readonly prisma: PrismaService) {}
+
+  async createAccommodation(createAccommodationBody: any) {
+    const newAccommodation = await this.prisma.accommodation.create({
```

Comment:
you should wrap all awaits to try catch block, because there are possible unexpected exceptions